components/NewWordForm/NewWordForm.js

```diff
@@ -0,0 +1,212 @@
+import styled from "styled-components";
+import { nanoid } from "nanoid";
+
+export default function NewWordForm({ onCreateNew }) {
+  async function handleSubmit(event) {
+    event.preventDefault();
+    const fields = event.target.elements;
+
+    const language = fields.queryLanguage.value.split("-");
+    const newWord = {
+      category: fields.category.value,
+      base: {
+        language: "english",
+        flag: "🇬🇧",
+        translation: fields.english.value,
+      },
+      query1: {
+        id: nanoid(),
+        language: language[1],
+        flag: language[0],
+        translation: fields.queryLanguage1.value,
+        gender: fields.gender.value,
+      },
+    };
+
+    onCreateNew(newWord);
+    console.log("new word: ", newWord);
+
+    event.target.reset();
+    fields.english.focus();
+  }
+
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <LabelEng htmlFor="english" name="english">
+        🇬🇧 english :
+      </LabelEng>
+      <InputEng
+        id="english"
+        name="english"
+        type="text"
+        maxLength={50}
+        placeholder="type word in english"
+        required
+      ></InputEng>
+
+      <LabelQuery htmlFor="queryLanguage1" name="queryLanguage1">
+        <select name="queryLanguage" id="queryLanguage">
+          <option selected disabled>
+            language
+          </option>
+          <option value="🇭🇷-croatian" name="croatian">
+            🇭🇷
+          </option>
+          <option value="🇨🇿-czech" name="czech">
+            🇨🇿
+          </option>
```

Comment:
Just as an idea: you could generate these options dynamically:

```jsx
languages.map(lang => <option value={lang.flag…} name={lang.name})
```
For that you would need an array of your languages. Not really necessary, but for your consideration ;)

Reply:
thanks for the suggestion! yes I can and I will - when I dynamically generate my categories array in another user story! :)